.github/workflows/phpunit.yml

```
@@ -20,23 +20,13 @@ jobs:
     strategy:
       matrix:
         drupal-version:
-          - '10.1'
+          - '10.2'
         php-version:
           - '8.2'
         experimental: [false]
         do-leniency: [true]
         patches: ["{}"]
```

Suspecting we might still want/need the patch (https://www.drupal.org/node/2845144 is still open/needs work), so probably need something like:

```suggestion
        patches: [
          |-
              {
                "drupal/core": {
                  "Referencable unpublished nodes": "https://www.drupal.org/files/issues/2024-01-07/2845144-78.patch"
                }
              }
        ]
```